frontend/views.py

```diff
@@ -71,28 +72,25 @@ def get_page(request, file_name):
 def redirect_images_to_media(request, image):
     return redirect(settings.MEDIA_URL + "images/" + image)
 
+@require_POST
 def submit_page_change(request):
 
-    if request.method == 'POST':
-
-        patch = get_diff(request.POST.get("page", ""),
-                         request.POST.get("content", ""))
-        form = PageEditForm({"page": request.POST.get("page", ""),
-                             "patch": patch,
-                             "comment": request.POST.get("descr", ""),
-                             "email": request.POST.get("email", ""),
-                             "date": datetime.datetime.now()})
-        if form.is_valid():
-            form.save()
-            return HttpResponse('')
-        else:
-            error_html = [ "<strong>{key}</strong>: {message}".format(key=key,
-                                                                      message=', '.join(message))
-                           for key, message in form.errors.items() ]
-            error_html = "<br>".join(error_html)
-            return HttpResponseForbidden(error_html)
-    else:
-        raise Http404
+    patch = get_diff(request.POST.get("page", ""),
+                     request.POST.get("content", ""))
+    form = PageEditForm({"page": request.POST.get("page", ""),
+                         "patch": patch,
+                         "comment": request.POST.get("descr", ""),
+                         "email": request.POST.get("email", ""),
+                         "date": datetime.datetime.now()})
+    if form.is_valid():
+        form.save()
+        return HttpResponse('')
+
+    error_html = [ "<strong>{key}</strong>: {message}".format(key=key,
+                                                              message=', '.join(message))
+                   for key, message in form.errors.items() ]
```

Comment:
That looks like a place where we'll have unicode/encoding error here :x 

But I guess we can wait to have that failed in prod, we'll see.